src/android/ContactAccessorSdk5.java

```diff
@@ -1631,24 +1631,41 @@ private void insertPhoto(ArrayList<ContentProviderOperation> ops,
     }
 
     /**
-       * Get an input stream based on file path or uri content://, http://, file://
-       *
-       * @param path
-       * @return an input stream
+     * Get an input stream based on file path or uri content://, http://, file://
+     *
+     * @param path path to file
+     * @return an input stream
      * @throws IOException
-       */
+     */
     private InputStream getPathFromUri(String path) throws IOException {
+        String ASSET_PREFIX = "file:///android_asset/";
```

Comment:
Please set this as a constant at the top, similarly to `EMAIL_REGEXP ` and `MAX_PHOTO_SIZE `.